### PR TITLE
[py] Bump to 1.0.9

### PR DESCRIPTION
**Pull request by LastMileNewGrad**
[py] Bump to 1.0.9

Republished AIConfig to 1.0.8 Changes Made Locally:
updated dependencies in pyproject.toml, should be good to go on that front.
(small) added an await statement that was missing for a callback
changes made locally
If any problems arise please let me know, I will fix asap.

1.0.9:

- Updated openai version to >1.0.0
- updated model parser and wrapper as well

### CODE REVIEW

**Review by saqadri — APPROVED** [review]

File: python/pyproject.toml [hunk 1]
```diff
@@ -26,6 +26,8 @@ dependencies = [
   "openai>=1.0.0",
   "python-dotenv",
   "huggingface_hub",
+  "result",
+  "nest_asyncio",
```
[the remark this answers is not in the document]
Comment (LastMileNewGrad) [reply python/pyproject.toml:30]: result is the library dependency we used for the `OK` and `error types.

The `nest_asyncio` is used in the OpenAI introspection wrapper to allow execution in Jupyter env

[the remark this answers is not in the document]
Comment (jonathanlastmileai) [reply python/pyproject.toml:30]: FWIW I dont think you need nest_asyncio with recent jupyter versions

[the remark this answers is not in the document]
Comment (jonathanlastmileai) [reply python/pyproject.toml:30]: Also @LastMileNewGrad why don't we add a changelog?

[the remark this answers is not in the document]
Comment (LastMileNewGrad) [reply python/pyproject.toml:30]: Where would the changelog go? I presume discussions would the best

**Comment by LastMileNewGrad** [comment]
rebase

**Pull request merged**